src/reducer.js

```diff
@@ -446,13 +451,13 @@ const createReducer = structure => {
         Object
           .keys(reducers)
           .reduce((accumulator, key) => {
-            const previousState = getIn(accumulator, key)
-            const nextState = reducers[ key ](previousState, action, getIn(state, key))
-            return nextState === previousState ?
-              accumulator :
-              setIn(accumulator, key, nextState)
-          },
-          this(state, action)))
+              const previousState = getIn(accumulator, key)
```

Comment:
No consciously, no. Was added by my IDE, and ESLint doesn't mind.

Personally, I think I prefer that wrapped continuations of parameters (in this case an arrow function) be indented....but not very strongly.

Thank you for the eagle eye, though. I'd be open to hearing arguments against it.